storyblok/storyblok-design-system
# refactor(sbTag): :recycle: SbTag component SFC refactor

## Pull request type

Jira Link: [INT](url) [Not applicable]




## How to test this PR



1. First build the Design System by running: `yarn storybook`
2. Check if the `<SbTag/>` component and all its variation are still working as previously
3. Verify all Unit Tests are still passing by running `yarn test:unit`

## What is the new behavior?



No new behavior has been added. The component instance creation has been refactored from a render function to a SFC (Single File Component).
A new unit test has been added to assert that when the component is instantiated without setting the `closable` prop to `true`. It should not have a `<SbIcon>` component child.

## Other information

N/A
